src/Drawer/Drawer.js

```diff
@@ -93,6 +98,8 @@ export default class Drawer extends Component {
       children,
       className,
       docked,
+      enterTransitionDuration,
```

Comment:
Didnt want to duplicate theese durations here (from Slider), but it turned out, Modal needs this durations to properly show/hide himself